images/linux/scripts/installers/aliyun-cli.sh

```diff
@@ -22,6 +22,6 @@ if ! command -v aliyun ; then
 fi
 
 # Document what was added to the image
-aliyun_version="$(aliyun --version | grep "Alibaba Cloud Command Line Interface Version" | cut -d " " -f 7)"
+aliyun_version="$(aliyun version)"
 echo "Lastly, documenting what we added to the metadata file"
 DocumentInstalledItem "Alibaba Cloud CLI ($aliyun_version)"
```

Comment:
@ideal, Thank you for the first contribution. Could you please change the title to `Alibaba Cloud CLI ($aliyun_version)` and  set the same value for Windows and Linux?

Reply:
OK. Do you mean it like this? https://github.com/actions/virtual-environments/pull/1150/commits/66515dc850149e09f9e9c882b81df0dd18ff70b9

Reply:
Thank you. It looks good to me.